foxpuppet/windows/__init__.py

```diff
@@ -3,5 +3,5 @@
 # You can obtain one at http://mozilla.org/MPL/2.0/.
 
 from foxpuppet.windows.base import BaseWindow  # noqa: F401
-from foxpuppet.windows.browser import BrowserWindow  # noqa: F401
+from foxpuppet.windows.browser.window import BrowserWindow  # noqa: F401
```

Comment:
@whimboo this is what we have already.. `from foxpuppet.windows import BrowserWindow` works exactly because of this line that you've commented on. We could move it to `foxpuppet/windows/browser/__init__.py` but we'd still also need `from foxpuppet.windows.browser import BrowserWindow` in this file. By making the change here, it's only one line.